Northstar.Client/mod/resource/northstar_client_localisation_english.txt

```diff
@@ -321,5 +321,18 @@ Press Yes if you agree to this. This choice can be changed in the mods menu at a
 		"INVALID_MASTERSERVER_TOKEN" 	"Invalid or expired masterserver token"
 		"JSON_PARSE_ERROR" 				"Error parsing json response"
 		"UNSUPPORTED_VERSION" 			"The version you are using is no longer supported"
+		
+		// Mod Settings
+		"MOD_SETTINGS"					"Mod Settings"
+		"NORTHSTAR_BASE_SETTINGS"		"Northstar Base Settings"
+		"ONLY_HOST_MATCH_SETTINGS"		"Only Host can change Private Match settings"
+		"ONLY_HOST_CAN_START_MATCH"		"Only Host can Start the Match"
+		"MATCH_COUNTDOWN_LENGTH"		"Private Match Countdown Duration"
+		"LOG_UNKNOWN_CLIENTCOMMANDS"	"Log Unknown Client Commands"
+		"DISALLOWED_TACTICALS"			"Prohibited Tacticals"
+		"TACTICAL_REPLACEMENT"			"Replacement Tactical"
+		"DISALLOWED_WEAPONS"			"Prohibited Weapons"
+		"REPLACEMENT_WEAPON"			"Replacement Weapon"
+		"SHOULD_RETURN_TO_LOBBY"		"Return To Lobby After Match End"
```

Comment:
Add support for other languages ? Probably should be a companion PR

Reply:
I think it'd be better for the English base strings to be left in so it'll be easier to translate to other languages.

Reply:
English should Def be left in, I just meant other langs should be added too

Reply:
I'll leave this in and if others want to change the contents, they are able to do so. This will have to be added sooner or later, so I'd rather do it now rather than have contributors change the codebase afterwards.